Added QuickAccessMenu

agaertner opened the pull request:
Allows to ping ones own truthful killproof from anywhere using ArcDpsService and Gw2Api.

agaertner committed:
Merge branch 'master' into dev

agaertner committed:
QuickAccessMenu

Allows to ping ones own killproof from anywhere using ArcDps and Gw2Api.

dlamkins commented on KillProofModule.cs:
```diff
@@ -182,7 +239,21 @@ protected override async Task LoadAsync()
         }
-
+        private void SendToChat(string message) {
+            System.Windows.Forms.Clipboard.SetText(message);
```
Oh, I would recommend you use the `ClipboardUtil`.  You can see an example of how it is used here:

https://github.com/blish-hud/Blish-HUD/blob/2a3b9c101d37d3571bb2f78f7f7e34303b2848a7/Blish%20HUD/Controls/TextInputBase.cs#L569-L574

We found out the hard way that setting the clipboard text is actually not always reliable and will sometimes throw an exception.  This will help prevent that from happening and once it completes you can confirm it went through successfully and then run that task below.

dlamkins replied:
You'll just need to just include the Nuget package that declares the reference for the library we use in the core application.

`Install-Package AsyncClipboardService -Version 1.7.1`

agaertner committed:
Added AsyncClipboardService

agaertner committed:
fix QuickAccessMenu blocking the render loop

when it pulls MyKillProof it blocked the render loop.

agaertner committed:
Fix 0 quantity on default selected token

When the panel built it showed 0 on the default selected token until the user manually switched from and back to it. By introducing a placeholder selected item at the construction we let it trigger our ValueChanged delegate through which we pull the correct value immediately when we assign the real default selected token.

agaertner commented:
The async change introduced a small issue I oversaw. This is fixed with https://github.com/blish-hud/KillProof-Module/pull/5/commits/ff8fd670f296ae2556e25ae5597005b23e1cf13a
